arch/arm/soc/atmel_sam0/samd/Kconfig.defconfig.series

```diff
@@ -0,0 +1,24 @@
+# Kconfig - Atmel SAMD MCU series configuration options
+*
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if SOC_SERIES_SAMD
+
+config SOC_SERIES
+	string
+	default samd
+
+config SOC_PART_NUMBER
+	string
+	default samd21g18a if SOC_PART_NUMBER_SAMD21G18A
+
```

Comment:
We should add here all other part numbers which were imported to ext/hal/atmel/asf/sam0/include/samd/

Reply:
That's a lot of part numbers :)

Done.